### ISSUE TITLE
BUG in Blade @if( != )

### ISSUE THREAD

**Issue by juancarlosso** [biati-digital/nova-php-cs-fixer#13]
The problem ocurs when i save my blade file.
There are a condition that brakes the format.

Here you have my code:
[Code Before Save](http://jasoluciones.info/temporal/bug1.png)

And this is the result:
[The result](http://jasoluciones.info/temporal/bug2.png)

Thanks, if you need something else to fix it, let me know.

**Comment by biati-digital** [comment]
Hi @juancarlosso please share the code so I can copy/paste it and make some tests.

**Comment by juancarlosso** [comment]
<div class="form-row">
      <div class="col-sm-2 mb-2">
       <label for="basicInput">Estatus</label>
       <select class="select2 form-control" name="estatus">
        @if(Auth::user()->perfil_id == 1)
        @foreach ($estados as $element)
        <option value="{{ $element->nombre }}"
         @if ($element->nombre != 'FINALIZADO' )
         {{ (($element->nombre == $ticket->estatus)||($element->nombre == old('estatus'))) ? 'selected' : '' }}>
         {{ $element->nombre }}
         @endif
        </option>
        @endforeach
        @else
        @foreach ($estados as $element)
        <option value="{{ $element->nombre }}"
         {{ (($element->nombre == $ticket->estatus)||($element->nombre == old('estatus'))) ? 'selected' : '' }}>
         {{ $element->nombre }}
        </option>
        @endforeach
        @endif
       </select>
       @error('tipo_solicitud')
       <span class="invalid-feedback" role="alert">
        <strong>{{ $message }}</strong>
       </span>
       @enderror
      </div>
      <div class="col-sm-8">
       <label for="basicInput">Comentarios del estatus</label>
       <input type="text"  class="form-control" name="comentario_estatus" value="{{ old('comentario_estatus') }}">
       <input type="hidden"  name="asunto" value="{{ $ticket->asunto ?? old('asunto') }}">
      </div>
      <div class="col-sm-2">
       <label style="color: white">Actualizar datos</label>
       <button type="submit" class="btn btn-block btn-primary pull-right"><i class="fa fa-floppy-o"></i> Guardar</button>
      </div>
     </div>

**Comment by biati-digital** [comment]
This should be fixed in the latest version 3.0.2, please update the extension and try it.

I do not use blade so one question I've is. Is the @endif correctly positioned in the following code?

```php
<option value="{{ $element->nombre }}"
     @if ($element->nombre != 'FINALIZADO' )
     {{ (($element->nombre == $ticket->estatus)||($element->nombre == old('estatus'))) ? 'selected' : '' }}>
     {{ $element->nombre }}
     @endif
```

This basically is formatted as

```php
<option value="{{ $element->nombre }}" @if ($element->nombre != 'FINALIZADO' )
         {{ (($element->nombre == $ticket->estatus)||($element->nombre == old('estatus'))) ? 'selected' : '' }}>
         {{ $element->nombre }}
     @endif
```

Like I said, I do not use blade so I'm not sure, it's just that it looks weird the `@endif` in that position.  js-beautify will not be able to indent correctly that code.

Just for the record, I try it with some other blade examples I found on the internet and the formatter was able to indent the code correctly. Anyway, thanks for the bug report.

**Comment by juancarlosso** [comment]
Everything is working fine ... thanks bro!
